src/camlsnark_c/camlsnark_linker_flags_gen.ml

```diff
@@ -17,9 +17,8 @@ let () =
         ; "-lstdc++" ]
     | "Linux" ->
         [ "-Wl,-E"
```

Comment:
Can you also replace the first arg for `"Darwin"` with `-lcamlsnark_c_stubs` to get the delicious space savings?

Reply:
Nice, done :)